config/loader.go

```diff
@@ -211,11 +212,17 @@ func (s *signalFxLoader) Forwarder(conf *ForwardTo) (protocol.Forwarder, error)
 		TraceSample:        conf.TraceSample,
 	}
 	if sfConf.TraceSample != nil {
-		sfConf.TraceSample.EtcdServer = conf.Server
 		sfConf.TraceSample.EtcdClient = conf.Client
 		sfConf.TraceSample.AdditionalDimensions = datapoint.AddMaps(conf.AdditionalDimensions, conf.TraceSample.AdditionalDimensions)
 		sfConf.TraceSample.ClusterName = conf.ClusterName
 	}
+	if conf.TraceDistributor != nil {
```

Comment:
You should protect against both being defined.

Reply:
agree